api/src/main/java/team/unnamed/creative/sound/SoundRegistry.java

```diff
@@ -84,6 +97,22 @@ public interface SoundRegistry extends ResourcePackPart, Namespaced, Examinable
         return soundRegistry(namespace, sounds);
     }
 
+    /**
+     * Creates a new registry from the given
+     * sounds
+     *
+     * @param namespace The sound registry namespace
+     * @param sounds    The registered sounds
+     * @return A new sound registry instance
+     * @since 1.6.0
+     * @deprecated Use {@link #soundRegistry(String, Collection)} instead
+     */
+    @Deprecated
+    @ApiStatus.ScheduledForRemoval(inVersion = "2.0.0")
+    static @NotNull SoundRegistry of(final @NotNull String namespace, final @NotNull Collection<SoundEvent> sounds) {
+        return soundRegistry(namespace, sounds);
+    }
+
```

Comment:
Remove this, we don't want to add deprecated methods

Reply:
oops, I didn't notice it was Deprecated